src/alexapi/triggers/platformtrigger.py

```diff
@@ -54,11 +54,11 @@ def platform_callback(self, platform_continuous_callback=None):
 				long_press_thread.start()
 
 	def continuous_callback(self):
-		if self._platform_continuous_callback:
-			return self._platform_continuous_callback()
-		else:
+		if not self._platform_continuous_callback:
 			return False
 
+		return self._platform_continuous_callback()
```

Comment:
Could be a one liner:

```
return self._platform_continuous_callback() if self._platform_continuous_callback() else False
```
From another perspective, why do we bother keeping `self._platform_continuous_callback` variable and not let platform decide what to do about it?